packages/circuit-ui/components/FieldAtoms/FieldValidationHint.tsx

```diff
@@ -15,15 +15,16 @@
 
 import { HTMLAttributes } from 'react';
 import { css } from '@emotion/react';
-import { Confirm, NotifyCircle, Alert } from '@sumup/icons';
 import isPropValid from '@emotion/is-prop-valid';
+import { Confirm, NotifyCircle, Alert } from '@sumup/icons';
 
 import styled, { StyleProps } from '../../styles/styled';
 import { typography } from '../../styles/style-mixins';
 
 type Color = 'alert' | 'notify' | 'confirm';
 
-export interface ValidationHintProps extends HTMLAttributes<HTMLSpanElement> {
+export interface FieldValidationHintProps
```

Comment:
I agree that the naming isn't ideal. I tried to convey this by naming it a "hint" instead of an "error", but I suppose that's still not generic enough.

Let's tackle this in the next PR as you suggested.